### PR TITLE
[javascript] Update benchmarks for fyrejet

### PR DESCRIPTION
**Pull request by schamberg97**
* Disables marketing `X-Powered-By` header
* Uses slightly faster version of `send` function to better showcase the underlying performance
* Updates uWebSockets.js

### CODE REVIEW

File: javascript/fyrejet-uwebsockets/app.js [hunk 1]
```diff
@@ -6,17 +6,18 @@ var app = fyrejet({
 });
 
 app.set("etag", false);
+app.set('x-powered-by', false)
 
 app.get("/", function (req, res) {
-  res.send("");
+  res.sendLite("");
```
Comment (waghanza) [inline javascript/fyrejet-uwebsockets/app.js:12]: is this intended to be used in **production** ?

I mean, what are the differences between `send` and `sendLite` ?

Comment (schamberg97) [reply javascript/fyrejet-uwebsockets/app.js:12]: Yes. Fyrejet is heavily based on restana with res.sendLite being the renamed restana’s res.send. 

The goal of the framework, however, is to provide Express-like API with significantly improved performance. So it also implements most of Express API, besides a few deprecated features.

 The default res.send is thus a highly modified function of the same name from express.js, which, in comparison to res.sendLite also provides some features related to ETag, caching, etc. 

TL;DR: difference between res.sendLite and res.send is that the former has less features, but is much faster

**Comment by schamberg97** [comment]
@waghanza meanwhile, are there any other frameworks that might need uWS to be updated?

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
@schamberg97 Some other frameworks uses `uwebsocket` indeed, but the version is upto them
for an example in `chubbyjs` the version of `uwebsocket` is handle by the framework itself
https://github.com/chubbyjs/chubbyjs-uwebsockets-psr-http-message-bridge/blob/4796d8b789a2128862047b6bc19dcc14853dd0a0/package.json#L56

btw, @dominikzogg, how can you ensure to use an up-to-date `uwebsocket` release ? (I'm not even sure it's a problem for `chubbyjs`)

**Comment by dominikzogg** [comment]
@waghanza ^1.2.1 upgrades to the same version
